build.gradle

```diff
@@ -16,6 +16,9 @@ buildscript {
         // Workaround for Lint + Retrolambda.
         classpath 'me.tatarka.retrolambda.projectlombok:lombok.ast:0.2.3.a2'
 
+        // Generates build info without breaking incremental builds
+        classpath 'hu.supercluster:paperwork-plugin:1.2.1'
```

Comment:
Now you can rebase on latest `master` and apply plugin version from `dependencies.gradle`!